concreate/schema/image_schema.yaml

```diff
@@ -38,7 +29,18 @@ map:
           value: {type: int, required: True}
           expose: {type: bool}
           description: {type: str}
-  workdir: {type: str}
+  run:
+    map:
+      workdir: {type: str}
+      user: {type: text}
+      cmd:
+          type: seq
+          sequence:
+          - type: str
```

Comment:
this should be text probably - we can have just numeric commands - especially if cmd is used as an argument for entrypoint?

Reply:
This was the case before, I didn't change this. If you think it requires change, please open a ticket.